setup.py

```diff
@@ -201,8 +188,7 @@ def read_version_py(infname):
     "Natural Language :: English",
     "Programming Language :: C",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 2",
-    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
```

Comment:
Are there any best practices for when to include minor version numbers too?

Reply:
:shrug: not sure anyone actually looks at these anymore, the `python_requires` is the important bit.